xAPI.md

```diff
@@ -1193,7 +1193,9 @@ Run-Time Environment. See [Appendix C](#AppendixC) for examples of each format.
 	</tr>
 	<tr>
 		<td>numeric</td>
-		<td>A range of numbers represented by a minimum and a maximum delimited by <code>:</code>.
+		<td>A range of numbers represented by a minimum and a maximum delimited by <code>[:]</code>. 
```

Comment:
We need to be more descriptive here. For instance, based on this it isn't clear how to indicate the range only has a lower or upper bound. Is it legal to just write "[:]" for every possible number? Et cetera. If we're going to incorporate SCORM we need to be precise to avoid many of the same problems.